src/main/java/com/amazonaws/cloudhsm/examples/ECImportKey.java

```diff
@@ -0,0 +1,118 @@
+import com.amazonaws.cloudhsm.jce.provider.attributes.KeyPairAttributesMapBuilder;
+
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+
+import javax.crypto.Cipher;
+
+/**
+ * Generate an EC Key Pair using BouncyCastle then import it into the HSM
+ */
+public class ECImportKey {
+    private static String helpString = "ECImportKey\n" +
+            "Generate an EC Key Pair using BouncyCastle then import it into the HSM\n";
+
+    public static void main(String[] args) throws Exception {
+        try {
+            Security.addProvider(new CloudHsmProvider());
+        } catch (IOException ex) {
+            System.out.println(ex);
+            return;
+        }
+
+        // First, generate a keypair with BouncyCastle to test import
+        Security.addProvider(new BouncyCastleProvider());
+        KeyPairGenerator generator =
+                KeyPairGenerator.getInstance(
+                        "EC", BouncyCastleProvider.PROVIDER_NAME);
+        generator.initialize(new ECGenParameterSpec("prime256v1"));
+        KeyPair bcEcKeyPair = generator.generateKeyPair();
+        System.out.println("Generated Bouncy Castle EC Key Pair");
+
+        ECPrivateKey bcEcPrivateKey = (ECPrivateKey) bcEcKeyPair.getPrivate();
+        ECPublicKey bcEcPublicKey = (ECPublicKey) bcEcKeyPair.getPublic();
+
+        // Import the EC Key Pair using the ECPrivateKeySpec and ECPublicKeySpec
+        // Keys will be generated with default attributes
+        // Reference: https://docs.aws.amazon.com/cloudhsm/latest/userguide/java-lib-attributes_5.html
+        ECPrivateKeySpec ecPrivateKeySpec =
+                new ECPrivateKeySpec(bcEcPrivateKey.getS(), bcEcPrivateKey.getParams());
+
+        ECPublicKeySpec ecPublicKeySpec =
+                new ECPublicKeySpec(bcEcPublicKey.getW(), bcEcPublicKey.getParams());
+
+        KeyFactory keyFactory =
+                KeyFactory.getInstance(
+                    "EC",
+                    CloudHsmProvider.PROVIDER_NAME);
+
+        ECPrivateKey hsmECPrivateKey = (ECPrivateKey) keyFactory.generatePrivate(ecPrivateKeySpec);
+        ECPublicKey hsmECPublicKey = (ECPublicKey) keyFactory.generatePublic(ecPublicKeySpec);
+        KeyPair hsmKeyPair = new KeyPair(hsmECPublicKey, hsmECPrivateKey);
+
+        System.out.println("Imported EC KeyPair using the ECPrivateKeySpec and ECPublicKeySpec");
+
+        // Import the EC Key Pair using KeyAttributesMap in order to specify custom key
+        // attributes such as setting TOKEN to true.
+
+        // Set attributes for EC public key
+        KeyAttributesMap publicKeyAttrsMap = new KeyAttributesMap();
+        publicKeyAttrsMap.put(KeyAttribute.LABEL, "ImportedECPublicKey");
+        publicKeyAttrsMap.put(KeyAttribute.EC_PARAMS, bcEcPublicKey.getParams());
+        publicKeyAttrsMap.put(KeyAttribute.EC_POINT, bcEcPublicKey.getW());
+        publicKeyAttrsMap.put(KeyAttribute.TOKEN, true);
```

Comment:
Can we make the `token` attribute `false`, just to prevent accumulation of keys for users testing the service.